package.json

```diff
@@ -50,5 +52,44 @@
     "bugs": {
         "url": "https://github.com/Microsoft/axe-sarif-converter/issues"
     },
-    "homepage": "https://github.com/Microsoft/axe-sarif-converter#readme"
+    "homepage": "https://github.com/Microsoft/axe-sarif-converter#readme",
+    "license-check-and-add-config": {
```

Comment:
Can we extract this config to a separate file? Its just better to have a single responsibility config file instead of adding it in the already verbose `package.json`. Just my opinion. If it can't be done, it can't  be done. 😄

Reply:
It's a limitation of license-check-and-add, unfortunately (https://github.com/awjh/license-check-and-add/issues/8 tracks a feature on their end allowing it)

Reply:
I agree with you removing it would be cleaner. Unfortunately, it is required by the library.